src/ekomark/plots.py

```diff
@@ -244,9 +249,16 @@ def save_operators_to_pdf(path, theory, ops, me: EKO, skip_pdfs, change_lab=Fals
 
         # plot the operators
         # it's necessary to reshuffle the eko output
-        for mu2 in me.mu2grid:
-            results = me[mu2].operator
-            errors = me[mu2].error
+        for mu2, op in me.items():
+            change_lab = False
+            if rotate_to_evolution_basis:
+                if not qed:
+                    op = manipulate.to_evol(op, source=True, target=True)
+                else:
+                    op = manipulate.to_uni_evol(op, source=True, target=True)
+                change_lab = True
```

Comment:
`change_lab ` is activated only if `rotate_to_evolution_basis` is, so can be removed

Reply:
done in https://github.com/NNPDF/eko/pull/292/commits/4699064f9e4c5e368e148f1afb9614e0e9908689